.github/workflows/enable-pack-build-and-test.yaml

```diff
@@ -0,0 +1,114 @@
+jobs:
+  enable_pack_build_and_test_workflow:
+    runs-on: ubuntu-20.04
+    name: Enable Pack Build and Test workflow
+
+    steps:
+      - name: Add checkout paths to env context
+        shell: bash
+        run: |
+          echo "CI_DIR=${{ github.workspace }}/ci" >> ${GITHUB_ENV}
+          echo "TOOLS_DIR=${{ github.workspace }}/exchange-tools" >> ${GITHUB_ENV}
+          echo "PACKS_PATH=${{ github.workspace }}/packs" >> ${GITHUB_ENV}
+
+      - name: Checkout ci repo
+        uses: actions/checkout@v2
+        with:
+          repository: StackStorm-Exchange/ci
+          ref: ${{ env.ci_branch }}
+          path: ci
+          fetch-depth: 1
+
+      - name: Checkout exchange-tools repo
+        uses: actions/checkout@v2
+        with:
+          repository: StackStorm-Exchange/exchange-tools
+          ref: ${{ env.exchange_tools_branch }}
+          path: exchange-tools
+          fetch-depth: 1
+
+      - name: Checkout Pack Repos and enable their Build and Test Workflow
+        shell: bash
+        env:
+          PACKS_ORG: ${{ env.packs_org }}
+          PACKS_PREFIX: ${{ env.pack_repo_prefix }}
+        run: |
+          mkdir -p ${PACKS_PATH}
+          cd ${PACKS_PATH}
+          source ${TOOLS_DIR}/functions.sh
+          for repo_name in $(_gh_list_repo_names ${PACKS_ORG} ${PACKS_PREFIX}); do
+            echo "::group::Clone ${PACKS_ORG}/${repo_name}"
+            gh repo clone "${PACKS_ORG}/${repo_name}"
+            if latestTag=$(git -C ${repo_name} describe --tags `git -C ${repo_name} rev-list --tags --max-count=1`); then
+              echo latestTag = $latestTag
+              git -C ${repo_name} checkout $latestTag -b latestTagBranch
+            fi
+            echo
+            echo "::endgroup::" # DELETED notices will not be folded to simplify scanning action output
+            if [[ -z "$latestTag" ]]; then
+              rm -rf ${repo_name}
+              echo "DELETED clone of ${PACKS_ORG}/${repo_name}: missing git tags"
+              echo "${PACKS_ORG}/${repo_name}" >> ${GITHUB_WORKSPACE}/bad_pack_repos
+            elif [[ ! -f "./${repo_name}/pack.yaml" ]]; then
+              rm -rf ${repo_name}
+              echo "DELETED clone of ${PACKS_ORG}/${repo_name}: missing pack.yaml"
+              echo "${PACKS_ORG}/${repo_name}" >> ${GITHUB_WORKSPACE}/bad_pack_repos
+            else
+              #
+              #  This is not a bad Pack, ensure the Build and Test workflow is active
+              #
+              WORKFLOW_PATH="repos/${PACKS_ORG}/${repo_name}/actions/workflows/build_and_test.yaml"
+              WORKFLOW_NAME="CI"
+              echo "::group::Ensure ${WORKFLOW_NAME} workflow for ${PACKS_ORG}/${repo_name} is active"
+              if gh api --silent --method GET ${WORKFLOW_PATH} 2>/dev/null; then
+                echo "GitHub ${WORKFLOW_NAME} workflow found for ${PACKS_ORG}/${repo_name}"
+                if [[ "$(gh api --method GET ${WORKFLOW_PATH} --jq .state)" != "active" ]]; then
```

Comment:
What if we skipped this check and just blindly enable it every time this workflow runs? That way, the 60d clock would reset every time this runs.

I wonder if it could also self-enable? So, every time it runs it hits the API to reset the clock on the CI repo as well.

Reply:
Oh, that's interesting and might work, have you found docs around what things specifically constitute repository activity? I have not ... well, other than the obvious (commits ... and I'm with you, I don't like dummy commits).

Reply:
I wish we knew what types of things constituted "repository activity" ... I still haven't managed to find specific information on this. I'm hoping it's more than just "commits against the main branch" (because that'd mean we'd have to do dummy commits into it). What's the monitoring story like for StackStorm, maybe an alternative to dummy commits into `master` would be to simply monitor *this* job's enabled or disabled status with a "daily dead man's switch" or something along those lines.